privaterelay/views.py

```diff
@@ -86,6 +86,12 @@ def profile(request):
         _handle_fxa_profile_change(fxa)
         now_has_premium = profile.has_premium
         newly_premium = not had_premium and now_has_premium
+        if newly_premium:
+            profile.date_subscribed = datetime.now(timezone.utc)
+            profile.save()
+        elif had_premium and not now_has_premium:
+            profile.date_unsubscribed = None
```

Comment:
```suggestion
            profile.date_subscribed = None
```

Reply:
I wonder if we should set `date_subscribed = None` at all. If we leave it as the last datetime that the user subscribed, we could do a query to find "churned" customers - i.e. (pseudo-code):
```
churned_users = SocialAccount.objects
    .exclude(extra_data__contains='premium-relay')
    .include(user_id__in=User.objects.exclude(
        profile__date_subscribed=None
    ))
```

Reply:
Hmm good point. I was thinking of people unsubscribing and later resubscribing, but of course just overwriting the field too. Maybe I should rename it to `date_last_subscribed?

Only risk is that we then really should be using `has_premium` to determine whether someone has Premium, i.e. that future contributors should not look at this field and think they can use it for determining whether someone has Premium.

Anyway, I removed it now.